aviary/subsystems/energy/battery_builder.py

```diff
@@ -37,23 +37,48 @@ def build_mission(self, num_nodes, aviary_inputs=None) -> om.Group:
         return battery_group
 
     def get_states(self):
-        state_dict = {Dynamic.Mission.CUMULATIVE_ELECTRIC_ENERGY_USED: {'fix_initial': True,
-                                                                        'fix_final': False,
-                                                                        'lower': 0.0,
-                                                                        'ref': 1e4,
-                                                                        'defect_ref': 1e6,
-                                                                        'units': 'kJ',
-                                                                        'rate_source': Dynamic.Mission.ELECTRIC_POWER_IN_TOTAL,
-                                                                        'input_initial': 0.0}}
+        # need to add subsystem name to target name ('battery.') for state due
+        # to issue where non aircraft or mission variables are not fully promoted
```

Comment:
I believe this is intended behavior to prevent dynamic variables from being promoted and connected across phases and to prevent promoting internal variables